src/forms/PartSellByPlanForm.php

```diff
@@ -24,6 +24,7 @@ public function rules()
             [['plan_id'], 'integer'],
             [['client_id', 'contact_id'], 'integer'],
             [['description'], 'string'],
+            [['ticket_id'], 'integer'],
```

Comment:
```suggestion
            [['reason'], 'string'],
```

YouTrack ticket number has letters.